transfermethodrepository/src/main/java/com/hyperwallet/android/ui/transfermethod/repository/TransferMethodRepositoryImpl.java

```diff
@@ -79,6 +79,8 @@ public void createTransferMethod(@NonNull final HyperwalletTransferMethod transf
     public void loadTransferMethods(@NonNull final LoadTransferMethodListCallback callback) {
 
         HyperwalletTransferMethodQueryParam queryParam = new HyperwalletTransferMethodQueryParam.Builder()
+                .limit(100)
```

Comment:
can we change this to non literal value? and create constant for example `DEFAULT_LIMIT` or `QUERY_PARAM_DEFAULT_LIMIT` your choice thanks

Reply:
and I think its also possible to add unit test, by running a mockwebserver and get request if limit = 100, let me know if you want to know how to do it

Reply:
Constant variable created and test case completed for bank account list increased